modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/build.gradle

```diff
@@ -0,0 +1,10 @@
+//
+// Check GETTING_STARTED.markdown#liferay.workspace.target.platform.version
+// for information regarding usage.
+//
+
+/*subprojects {
```

Comment:
Lets remove this comment and instead add this section of code to GETTING_STARTED.